README.md

```diff
@@ -0,0 +1,86 @@
+  - El usuario podrá registrarse utilizando nombre, correo y clave ó una cuenta de Google
+  - Los datos de registro deben guardarse en firebase
+  - El formulario de registro debe contar con validaciones
+  - El diseño del formulario debe apegarse al proporcionado
+  
+- Permitir a los usuarios registrados iniciar sesión:
+  - El usuario podrá iniciar sesión utilizando correo y clave o su cuenta de Google, dependiendo de la forma en la que realizó su registro.
+  - El inicio de sesión debe realizarse accediendo o verificando los datos almacenados en firebase.
+  - El formulario de inicio de sesión debe contar con validaciones
+  - El diseño del formulario debe apegarse al proporcionado
+  
+- Permitir a los usuarios ver las reservaciones de sus vuelos:
+  - El usuario podrá ver un listado con la información de sus reservaciones de vuelos
+  - La información debe recuperarse desde firebase
+  - El diseño del listado debe apegarse al proporcionado
+  
+- Permitir agregar nuevas reservaciones
+  - El usuario deberá indiciar el origen, destino, fecha y número de pasajeros
+  - La información debe guardarse en firebase
+  - El diseño para la nueva reservación debe apegarse al proporcionado
+  
+ Algunos de los requerimientos sun un tanto genéricos, es decir no brindan el suficiente detalle, el tratamiento que se debe dar al proyecto es el de un MVP, es decir no es necesario que el funcionamiento o el cumplimiento a los requerimientos sea el más complejo, que cumpla con lo mínimo necesario para que cumpla su función
+  
+## Requerimientos no-funcionales
+ - Base de datos
+   - La información se guardará en firebase
+ - Calidad
+   - Utilizar un estilo de código estandarizado (revisado por Eslint)
+   - Incluir pruebas unitarias
+   - Puntuación **A** obtenida en CodeClimate
+- Ejecución 
+   - Puede ejecutarse en Android o iOs
+- Diseño
+   - Debe ser lo más cercano posible al proporcionado
+- Código fuente
+   - Orientado a Objetos
+   - Métodos pequeños
+   - Aplicar los principios [SOLID](https://blog.usejournal.com/how-to-apply-solid-principles-in-react-applications-6c964091a982)
+ - Manejo del estado
+   - Redux para el manejo del estado de la App
+  
+## Diseño
+En la carpeta [img](/img) de este repositorio se encuentra el diseño solicitado en formato PNG e [InVision](https://www.invisionapp.com/).
+  
+## Entregable
+- Código fuente en Github (en la rama master)
+- Incluir en el repositorio la puntuación obtenida (badge) en CodeClimate
+- Documentar en [este archivo](setup/README.md) los pasos necesarios para ejecutar la App
+- [Los commits deben se significativos](https://medium.com/better-programming/you-need-meaningful-commit-messages-d869e44e98d4)
+
+
+## Setup
+[Aquí se describen los pasos necesarios para ejecutar y probar este proyecto](setup/README.md)
+
+## Recursos
+[Para apoyo de tus actividades utiliza esta lista de recursos recomendados](https://github.com/bright-coders/commons/tree/master/topics/resources-react-native)
+  
```

Comment:
Why is the readme file in this PR?

Reply:
I deleted it by mistake some time ago, I just wanted to restore it so I added it to a PR